components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/internal/EntitlementServiceComponent.java

```diff
@@ -307,6 +317,36 @@ protected void activate(ComponentContext ctxt) {
         }
     }
 
+    /**
+     * Adds policy files with unique policyIDs to the registry.
+     *
+     * @param policyIdList
+     * @param fileList
+     * @return
+     * @throws IOException
+     */
```

Comment:
Complete the javadoc with descriptions

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3110